.github/workflows/release.yml

```diff
@@ -38,6 +39,10 @@ jobs:
                   registry-url: https://registry.npmjs.org
                   node-auth-token: ${{ secrets.NPM_TOKEN }}
 
+            # TODO Trusted publishing (OIDC) requires npm CLI version 11.5.1 or later.
```

Comment:
Why did we do this instead of just updating the node in the package.json?

Reply:
So you are saying that even though we are on the latest node, the latest node does not come with the latest npm?